roles/openshift_health_checker/openshift_checks/logging/elasticsearch.py

```diff
@@ -40,8 +33,10 @@ def run(self, tmp, task_vars):
         # TODO(lmeyer): run it all again for the ops cluster
         return {"failed": False, "changed": False, "msg": 'No problems found with Elasticsearch deployment.'}
 
-    def _not_running_elasticsearch_pods(self, es_pods):
+    def not_running_elasticsearch_pods(self, es_pods):
         """Returns: list of running pods, list of errors about non-running pods"""
+        # FIXME: the docstring above is clearly wrong.
+        # I don't understand this method enough to fix the doscstring.
```

Comment:
Docstring should say list of NON running pods.

Normally I don't want to do anything about pods that aren't running as they don't hurt anything and could be transient. What I want to do is filter them out so we don't think we have running pods when we don't. However I probably came to that conclusion after writing the initial draft of this method...